client/mysqladmin.cc

```diff
@@ -1351,7 +1351,8 @@ static void usage(void)
 
 static int drop_db(MYSQL *mysql, const char *db)
 {
-  char name_buff[FN_REFLEN+20], buf[10];
+  const size_t name_buf_len= FN_REFLEN+20;
```

Comment:
As long as you're removing/updating mysterious constants… any idea where the `+20` and the `10` come from, here?

Reply:
20 is just enough to fit the ``drop database `...` `` prefix around the string, and I figure 10 is considered enough to fit user input. Added defines for these.